gravitee-policy-quota/src/main/resources/schemas/schema-form.json

```
@@ -62,17 +62,37 @@
                 },
                 "periodTime": {
                     "type": "integer",
-                    "title": "Time duration",
-                    "default": 1
+                    "title": "Static duration",
+                    "description": "A numeric value will be preferred over dynamic expression when greater than 0.",
```

```suggestion
                    "description": "If defined, will be used instead of 'Dynamic time duration'",
```
What do you think of something like this?